include/gnc/constants.hpp

```diff
@@ -0,0 +1,54 @@
+/** @file gnc/constants.hpp
+ *  Autocoded constants header file. See tools/constants_generator.py for more
+ *  information. */
+
+#ifndef GNC_CONSTANTS_HPP_
+#define GNC_CONSTANTS_HPP_
+
+#include "config.hpp"
+
+#include <lin/core/matrix/matrix.hpp>
+#include <lin/core/vector/vector.hpp>
+
+namespace gnc {
+namespace constant {
+
+GNC_TRACKED_CONSTANT(constexpr static double, pi, 3.141592653589793);
+
+GNC_TRACKED_CONSTANT(constexpr static double, two_pi, 2.0 * pi);
+
+extern unsigned short init_gps_week_number;
```

Comment:
Why are some of these extern and some constexpr static?

Reply:
I think some are extern so that their values can be controlled by FSW (and therefore by the ground, during mission operation.)